server/src/connectionHandlers/onCompletion.ts

```diff
@@ -34,6 +36,54 @@ export function onCompletionHandler (textDocumentPositionParams: TextDocumentPos
     return []
   }
 
+  // overrides
+  if (word === ':') {
+    const wordBeforeIsIdentifier = analyzer.isIdentifier({
+      ...textDocumentPositionParams,
+      position: {
+        line: textDocumentPositionParams.position.line,
+        // Go two character back as one character back is ':'
+        character: Math.max(textDocumentPositionParams.position.character - 2, 0)
+      }
+    })
+    if (wordBeforeIsIdentifier) {
+      const overridesCompletionItems: CompletionItem[] = OVERRIDES.map(keyword => {
+        return {
+          label: keyword,
+          kind: CompletionItemKind.Keyword
+        }
+      })
+
+      return overridesCompletionItems
+    } else {
```

Comment:
unnecessary else

Reply:
the parent `if` block is entered when a `:` is typed, I don't want to show suggestions when it is not following an `identifier`.